model/group.h

```diff
@@ -219,7 +219,7 @@ namespace minsky
     void moveContents(Group& source);     
 
     /// returns which I/O region (x,y) is in if any
-    struct IORegion {enum type {none,input,output};};
+    struct IORegion {enum type {none,input,output,header,footer};};
```

Comment:
Why are you adding this here? It's not wrong, I just don't understand the motivation.